sdk/spring/spring-cloud-azure-appconfiguration-config-web/src/main/java/com/azure/spring/cloud/config/web/implementation/pullrefresh/AppConfigurationEventListener.java

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.azure.spring.cloud.config.web.pullrefresh;
+package com.azure.spring.cloud.config.web.implementation.pullrefresh;
 
-import static com.azure.spring.cloud.config.web.AppConfigurationWebConstants.ACTUATOR;
-import static com.azure.spring.cloud.config.web.AppConfigurationWebConstants.APPCONFIGURATION_REFRESH;
-import static com.azure.spring.cloud.config.web.AppConfigurationWebConstants.APPCONFIGURATION_REFRESH_BUS;
+import static com.azure.spring.cloud.config.web.implementation.AppConfigurationWebConstants.ACTUATOR;
+import static com.azure.spring.cloud.config.web.implementation.AppConfigurationWebConstants.APPCONFIGURATION_REFRESH;
+import static com.azure.spring.cloud.config.web.implementation.AppConfigurationWebConstants.APPCONFIGURATION_REFRESH_BUS;
```

Comment:
Line 20, if it's already defined as a bean, then there's no need to add the `@Component` annotation here.